pywwise/ak/wwise/core/profiler.py

```diff
@@ -1,6 +1,20 @@
+from dataclasses import fields
+from typing import Any
+
```

Comment:
Are these needed? `fields` seems a little out of place here (still resding the code at the time. If you are using `Any`, do `Any as _Any`.